features/checkout/addressing_order/changing_address_during_checkout.feature

```diff
@@ -0,0 +1,20 @@
+@checkout
+Feature: Changing address during checkout
+    In order to place an order with the correct address
+    As a Visitor
+    I want to be able to change the address during checkout
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "T-shirt banana" priced at "$12.54"
+
+    @api
+    Scenario: Changing address
```

Comment:
As this scenario is not checking an implemented improvement, are we sure we do not have a similar scenario, maybe not implemented in `@api` context? I just want to avoid the duplication

Reply:
This one looks similar: `features/checkout/addressing_order/returning_to_addressing_step_and_changing_address_data.feature`

Reply:
@Tomanhez From what I can see you deleted your scenario and didn't implement the one mentioned by me, why?

Reply:
I understood that this scenario is redundant in favor of above scenario

Reply:
But the above scenario is not implemented in `@api` context